source/main.c

```diff
@@ -275,6 +358,7 @@ int main()
         DCStoreRange((void *) STUB_ADDR, stub_size);
 
         SYS_ResetSystem(SYS_SHUTDOWN, 0, FALSE);
+        // How to pass argv?
         SYS_SwitchFiber((intptr_t) dol, 0,
                         (intptr_t) NULL, 0,
```

Comment:
You pass argv here as a string with NUL delimiters followed by its length.

Reply:
Sorry, you'll have to be more specific for me. This is the only documentation I could find on this function and I don't know which parameter accepts the string and length:

https://github.com/extremscorner/libogc2/blob/master/gc/ogc/system.h#L354

Reply:
`SYS_SwitchFiber` is irrelevant. It's the ABI of the relocation stub that matters.

It expects a pointer to argv in r5 (arg2) and the length of argv in r6 (arg3).

Reply:
Thanks for the help!